games-arcade/freegish/freegish-1.0.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="A free version of the game Gish"
+DESCRIPTION="FreeGish is an open-source, freely redistributable version of Gish, a \
+physics-based platforming game featuring a ball of tar."
+HOMEPAGE="http://github.com/freegish/freegish"
+COPYRIGHT="2012-2014 Joris van de Donk"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/freegish/freegish/archive/f53d9153.zip"
+CHECKSUM_SHA256="2d5e0f9d24ddc648e4cdf45adf50043ec13ecd991da61166d670f4be74447665"
+PATCHES="freegish-1.0.patchset"
+
+ARCHITECTURES="?x86 x86_gcc2 ?x86_64"
+
+PROVIDES="
```

Comment:
freegish is the whole package name here while app:FreeGish is the binary it contains. The idea is that you should be able to use `pkgman install freegish` as well as `pkgman install app:FreeGish`.